docs/src/proposals/big-transactions.md

```diff
@@ -0,0 +1,186 @@
+# Big Transactions
```

Comment:
nits: naming bike shedding: Extended Transactions? Compressed Transactions? Page-Indexed Transactions? or Transaction v2?

Reply:
Changed to Page-Indexed Transactions

Reply:
Updated to Transactions v2